crates/api_crud/src/community/update.rs

```diff
@@ -51,6 +52,10 @@ pub async fn update_community(
     is_valid_body_field(desc, false)?;
   }
 
+  if data.visibility == Some(lemmy_db_schema::CommunityVisibility::Private) {
+    is_admin(&local_user_view)?;
+  }
```

Comment:
This check is used multiple times now, so its probably better to extract it into a common utils function.

Reply:
Done although there are only two uses.